app/invocation/invocation_build_logs_card.tsx

```diff
@@ -1,14 +1,137 @@
 import React from "react";
-import InvocationModel from "./invocation_model";
+import { eventlog } from "../../proto/eventlog_ts_proto";
+import { invocation } from "../../proto/invocation_ts_proto";
+import capabilities from "../capabilities/capabilities";
+import errorService from "../errors/error_service";
+import rpcService from "../service/rpc_service";
 import { TerminalComponent } from "../terminal/terminal";
+import { BuildBuddyError } from "../util/errors";
+import InvocationModel from "./invocation_model";
 
 interface Props {
   model: InvocationModel;
   expanded: boolean;
 }
 
-export default class BuildLogsCardComponent extends React.Component {
-  props: Props;
+interface State {
+  consoleBuffer?: string;
+  nextChunkId?: string;
+  fetchedFirstChunk?: boolean;
+}
+
+const POLL_TAIL_INTERVAL_MS = 1_000;
+const MAX_INITIAL_LINES = 100_000;
+
+const InvocationStatus = invocation.Invocation.InvocationStatus;
+
+export default class BuildLogsCardComponent extends React.Component<Props, State> {
+  state: State = {
+    consoleBuffer: "",
+    nextChunkId: "",
+    fetchedFirstChunk: false,
+  };
+
+  private pollTailTimeout: number | null = null;
+
+  constructor(props: Props) {
```

Comment:
Can we just delete this constructor? This check doesn't do much and doesn't catch the case where the model changes.